src/index.tsx

```diff
@@ -298,7 +304,12 @@ export class App extends React.Component<any, AppState> {
       !event.metaKey &&
       this.state.draggingElement === null
     ) {
-      this.setState({ elementType: findShapeByKey(event.key) });
+      if (shape !== "text") {
+        document.documentElement.style.cursor = CURSOR_TYPE.CROSSHAIR;
+      } else {
+        document.documentElement.style.cursor = CURSOR_TYPE.TEXT;
+      }
```

Comment:
I'd personally swap the conditional, to make it clearer (keep the logic close to the branch the condition tests for):

```js
if (shape == "text") {
    document.documentElement.style.cursor = CURSOR_TYPE.TEXT;
} else {
    document.documentElement.style.cursor = CURSOR_TYPE.CROSSHAIR;
}
```

Reply:
@dwelle makes sense, thanks!